ci/jenkins/pipelines/skuba-cluster-upgrade-plan-1.14-nightly.Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+/**
+ * This pipeline verifies skuba cluster upgrade plan
+ */
+
+pipeline {
+   agent { node { label 'caasp-team-private' } }
+
+   environment {
+        OPENSTACK_OPENRC = credentials('ecp-openrc')
+        TERRAFORM_STACK_NAME = "${JOB_NAME}-${BUILD_NUMBER}"
+        GITHUB_TOKEN = credentials('github-token')
+        VMWARE_ENV_FILE = credentials('vmware-env')
```

Comment:
You will also need the `WORKSPACE` one, no?

Reply:
not sure... `WORKSPACE` should always be set/available by default i think